Sprint_4/src/test/java/ru/yandex/praktikum/scooter/pageobjects/ScooterHomePage.java

```diff
@@ -0,0 +1,136 @@
+package ru.yandex.praktikum.scooter.pageobjects;
```

Comment:
Необходимо исправить: В пакете src.**test**.java должны быть пакеты классов с кейсами. Остальные пакеты, такие как pageobgects, следует располагать в src.**main**.java.

Reply:
!!!!!!! Николай, не могу выполнить push  ошибка:
error: failed to push some refs to https:///.....Sprint_4.git
hint: Updates were rejected because the remote contains work that you do not have locally. This is usually caused by another repository pushing to the same ref. You may want to first integrate the remote changes <e.g. 'git pull ...' before pushing again. See the 'Note about fast-forwards' in 'git push -help'